topics/go/language/variables/exercises/exercise1/exercise1.go

```diff
@@ -19,9 +19,12 @@ func main() {
 	var legal bool
 
 	// Display the value of those variables.
-	fmt.Println(age)
-	fmt.Println(name)
-	fmt.Println(legal)
+	fmt.Printf(
```

Comment:
Revert all of the changes in this file. This is the first exercise people go through and we are just having them print the values. We want to start small so they are comfortable just building/running a Go program.

Reply:
Agree